static/templates/settings/organization-settings-admin.handlebars

```diff
@@ -133,6 +134,16 @@
                     {{/each}}
                 </select>
             </div>
+            <div class="input-group">
+                <label class="checkbox">
+                    <input type="checkbox" id="id_realm_send_welcome_emails" name="realm_send_welcome_emails"
+                           {{#if realm_send_welcome_emails}}checked="checked"{{/if}} />
+                    <span></span>
+                </label>
+                <label for="id_realm_send_welcome_emails" id="id_realm_send_welcome_emails_label" class="inline-block">
+                    {{t "Send emails to new users explaining how to use Zulip" }}
```

Comment:
This should stay "Introductory emails explaining to new users how Zulip works"

Reply:
Actually I'm not sure about this text; @rishig maybe worth chatting a bit about this?

Reply:
yeah, we can chat on czo when we're next online. I'm also not thrilled with this text.

Reply:
What should I change it to?